### PR TITLE
QUA-958: Push images to Dockerhub instead of GCR and update CircleCI config to version 2.1

### PR DESCRIPTION
**Pull request by dantevvp**
Deprecate pushing of the image to GCR in favor of pushing to Dockerhub.

Also, move the CircleCI config to version 2.1

### CODE REVIEW

**Review by larkinscott — APPROVED** [review]

**Comment by dantevvp** [comment]
@larkinscott i think i can't merge this because i dont have write access to the repo. Can you merge this whenever you have a second?

**Pull request merged**
